.gitignore

```diff
@@ -109,6 +109,9 @@ dist
 # TernJS port file
 .tern-port
 
+# VSCode configs
+.vscode
```

Comment:
@hoang-dao .vscode is added to the git on the purpose of suggesting configuration of vs. There is no mistake here.

Reply:
I think we shouldn't commit this settings to our project, VSCode settings is personalized configs, it's based on the style that the developer want to work with, forcing anyone to use a specific configs is not a good idea.

Reply:
I basically agree with @hoang-dao as members can use `.git/info/exclude` to hide their personal settings but if the team decided to use VSCode or whatever as an official IDE for some convention I think it's fine to include it.